lemminx-liberty/src/test/java/io/openliberty/LibertyWorkspaceTest.java

```diff
@@ -30,4 +30,22 @@ public void testReadDevcMetadata() throws URISyntaxException {
         // assertEquals("liberty-dev", libertyWorkspace.getContainerName());
         // assertTrue(libertyWorkspace.isContainerAlive());
     }
+
+    @Test
+    public void testBackslashConfigDetection() throws IOException {
+        // run test on Windows
+        if (File.separator.equals("/")) {
+            return;
+        }
+
+        File mockXML = new File("src/test/resources/configDropins/defaults/my.xml");
+        String filePathString = mockXML.getCanonicalPath();
+        URI filePathURI = mockXML.toURI();
+
+        assertFalse(LibertyUtils.isConfigDirFile(filePathString));
+        assertTrue(LibertyUtils.isConfigDirFile(filePathURI.toString()));
+        // mock replacement
+        filePathString = filePathString.replace("\\", "/");
+        assertTrue(LibertyUtils.isConfigDirFile(filePathString));
```

Comment:
This confuses me. Why are you replacing backslash with forward slash before calling `isConfigDirFile`? Seems like we should be testing that if the input has backslashes, it can still deal with it correctly.

Reply:
This test is a bit of a hack I admit. I didn't want to setup a mock Liberty workspace and project manager to check this simple behavior. 
This test seeks to prove two things when a file is read on Windows:
- the URI is handled correctly without edit when the constants are set to strings with forward slashes
- `getCanonicalPath` will fail `isConfigDirFile` when read on Windows until handled with the replace